classes/WP_CLI/ActionScheduler_WPCLI_Scheduler_command.php

```diff
@@ -5,6 +5,37 @@
  */
 class ActionScheduler_WPCLI_Scheduler_command extends WP_CLI_Command {
 
+	/**
+	 * Force tables schema creation for Action Scheduler
+	 *
+	 * ## OPTIONS
+	 *
+	 * @param array $args Positional arguments.
+	 * @param array $assoc_args Keyed arguments.
+	 * @throws \WP_CLI\ExitException When an error occurs.
+	 *
+	 * @subcommand fix-schema
+	 */
+	public function fix_schema( $args, $assoc_args ) {
+		$schema_classes = array( 'ActionScheduler_LoggerSchema', 'ActionScheduler_StoreSchema' );
+
+		foreach ( $schema_classes as $classname ) {
+			if ( is_subclass_of( $classname, 'ActionScheduler_Abstract_Schema' ) ) {
```

Comment:
```suggestion
		$schema_classes = array( ActionScheduler_LoggerSchema::class, ActionScheduler_StoreSchema::class );

		foreach ( $schema_classes as $classname ) {
			if ( is_subclass_of( $classname, ActionScheduler_Abstract_Schema::class ) ) {
```

Optional—but this would provide a couple of benefits especially in IDEs (easy navigation to the actual classes, and during future refactoring, etc).